package.json

```diff
@@ -28,7 +28,7 @@
     "lint:fix": "eslint --fix .",
     "prepublishOnly": "npm run clean",
     "proxydirs": "node scripts/proxy-dirs.js",
-    "prepack": "run-s build proxydirs",
+    "prepack": "npm i && run-s build proxydirs",
```

Comment:
This would also run `npm install` prior to a `npm publish` I think? Not sure if that's necessarily a good idea (a dependency might get updated that causes something to break that you didn't catch while testing, for example).

Reply:
Yes, it would. There doesn’t seem to be a simple way to do this only for git dependency installs without potentially interfering with the actual build/publish steps, so perhaps it’s best just to accept that git installs aren’t supported for now and encourage linking for local development where necessary.